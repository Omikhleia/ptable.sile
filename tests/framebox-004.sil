\begin[papersize=a6, class=resilient.book]{document}
\use[module=packages.framebox]
\use[module=packages.struts]
\nofolios
\neverindent
% Test case: migrating content in framebox and colleagues.

1. \framebox{square box\footnote{square}.}

\medskip
2. \roundbox{rounded box\footnote{rounded}.}

\medskip
3. \roughbox{rough box\footnote{rough}.}

\medskip
4. \framebox{\parbox[width=60pt]{\raggedright{Lorem ipsum\footnote{framed parbox} dolor sit amet.}}}


\end{document}
