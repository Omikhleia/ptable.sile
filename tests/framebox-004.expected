Set paper size 	297.6377985	419.5275636
Begin page
Mx 	29.7638
My 	58.2336
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	20 w=4.6924	(1)
Mx 	34.4562
T	17 w=2.2900	(.)
Draw SVG	58.2336	54.7651	13.9121	1.0000	0 4.34375 54.76515 13.91211 re 0 0 0 RG 1 1 1 rg 0.4 w B
Mx 	41.4072
T	86 84 88 68 85 72 w=27.4805	(square)
Mx 	71.5486
T	69 82 91 w=15.1367	(box)
Mx 	86.7853
My 	55.0548
Set font 	Gentium Plus;6.6;600;;normal;+lnum -onum;;LTR
T	20 w=3.0970	(1)
Mx 	89.8823
My 	58.2336
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	17 w=2.2900	(.)
Mx 	29.7638
My 	79.1457
T	21 w=4.6924	(2)
Mx 	34.4562
T	17 w=2.2900	(.)
Draw SVG	79.1457	62.1081	11.7637	1.0000	5 2.19531 m 57.10809 2.19531 l 59.86951 2.19531 62.10809 4.43389 62.10809 7.19531 c 62.10809 8.95898 l 62.10809 11.72041 59.34667 13.95898 57.10809 13.95898 c 5 13.95898 l 2.23858 13.95898 0 11.19756 0 8.95898 c 0 7.19531 l 0 4.43389 2.76142 2.19531 5 2.19531 c 0 0 0 RG 1 1 1 rg 0.4 w B
Mx 	41.4063
T	85 82 88 81 71 72 71 w=34.8242	(rounded)
Mx 	78.8907
T	69 82 91 w=15.1367	(box)
Mx 	94.1274
My 	75.9670
Set font 	Gentium Plus;6.6;600;;normal;+lnum -onum;;LTR
T	21 w=3.0970	(2)
Mx 	97.2244
My 	79.1457
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	17 w=2.2900	(.)
Mx 	29.7638
My 	97.1457
T	22 w=4.6924	(3)
Mx 	34.4562
T	17 w=2.2900	(.)
Draw SVG	97.1457	47.9680	10.0098	1.0000	-0.13561 0.11142 m 16.56698 0.60883 33.4766 -0.52815 51.37254 1.79388 c -2.45688 -0.50678 m 10.13683 -0.15391 24.09543 -0.10608 50.77668 0.69496 c 50.05 0.92449 m 48.58498 4.64443 48.93989 6.80389 49.90324 13.63573 c 49.93136 -0.09237 m 50.2499 6.11378 49.60792 12.02526 50.41794 14.27384 c 50.62865 14.20141 m 31.30556 13.88763 13.04083 15.86872 -3.56439 16.38726 c 50.44463 13.7214 m 29.13976 14.97133 10.69425 14.57942 -2.3218 14.76761 c -3.24975 13.25981 m -2.4909 11.7211 -0.89442 8.04058 -1.82599 1.23074 c -1.42363 14.27356 m -1.9158 11.23531 -1.49105 7.11636 -1.66845 1.09527 c 0 0 0 RG 0.4 w S
Mx 	39.4079
T	85 82 88 74 75 w=24.6826	(rough)
Mx 	66.7522
T	69 82 91 w=15.1367	(box)
Mx 	81.9889
My 	93.9670
Set font 	Gentium Plus;6.6;600;;normal;+lnum -onum;;LTR
T	22 w=3.0970	(3)
Mx 	85.0859
My 	97.1457
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	17 w=2.2900	(.)
Mx 	29.7638
My 	115.1457
T	23 w=4.6924	(4)
Mx 	34.4562
T	17 w=2.2900	(.)
Draw SVG	115.1457	64.0000	34.3477	1.0000	0 26.19531 64 34.34766 re 0 0 0 RG 1 1 1 rg 0.4 w B
Mx 	41.4061
T	47 82 85 72 80 w=26.4014	(Lorem)
My 	127.1457
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	66.7063
My 	123.9670
Set font 	Gentium Plus;6.6;600;;normal;+lnum -onum;;LTR
T	23 w=3.0970	(4)
Mx 	72.4459
My 	127.1457
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	41.4061
My 	139.1457
T	86 76 87 w=10.0146	(sit)
Mx 	54.0634
T	68 80 72 87 w=20.6836	(amet)
Mx 	74.7470
T	17 w=2.2900	(.)
Mx 	29.8638
My 	323.0245
Set font 	Gentium Plus;5.94;600;;normal;+lnum -onum;;LTR
T	20 w=2.7873	(1)
Mx 	35.0294
My 	325.8853
Set font 	Gentium Plus;9;400;;normal;;;LTR
T	86 84 88 68 85 72 w=24.7324	(square)
Mx 	29.8638
My 	334.3770
Set font 	Gentium Plus;5.94;600;;normal;+lnum -onum;;LTR
T	21 w=2.7873	(2)
Mx 	35.0294
My 	337.2378
Set font 	Gentium Plus;9;400;;normal;;;LTR
T	85 82 88 81 71 72 71 w=31.3418	(rounded)
Mx 	29.8638
My 	345.7295
Set font 	Gentium Plus;5.94;600;;normal;+lnum -onum;;LTR
T	22 w=2.7873	(3)
Mx 	35.0294
My 	348.5904
Set font 	Gentium Plus;9;400;;normal;;;LTR
T	85 82 88 74 75 w=22.2144	(rough)
Mx 	29.8638
My 	357.0821
Set font 	Gentium Plus;5.94;600;;normal;+lnum -onum;;LTR
T	23 w=2.7873	(4)
Mx 	35.0294
My 	359.9429
Set font 	Gentium Plus;9;400;;normal;;;LTR
T	73 85 68 80 72 71 w=26.5605	(framed)
Mx 	63.9852
T	83 68 85 69 82 91 w=26.0859	(parbox)
End page
Finish
